docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
             - POSTGRES_USER=postgres
             - POSTGRES_PASSWORD=password
         ports:
-            - "5432:5432"
+            - "5433:5432"
```

Comment:
would change this back to 5432

in the future you can also just not stage the docker-compose.yml file before you commit if it has a change like this